sweetest/src/main/java/com/mysugr/sweetest/framework/coroutine/AssertSequence.kt

```diff
@@ -0,0 +1,15 @@
+package com.mysugr.sweetest.framework.coroutine
+
+suspend fun assertSequence(block: suspend AssertSequence.() -> Unit) {
+    block(AssertSequence())
+}
+
+class AssertSequence {
+    private var currentSequenceIndex = 0
+    fun expect(sequenceIndex: Int) {
```

Comment:
it doesn't have the same signature as [kotlin's `expect`](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/expect.html) but it confused me for a second. i don't know what is better here: simplicity (`expect`) or more specificity (`expectSequence`)?